city_metrix/layers/glad_lulc.py

```diff
@@ -0,0 +1,126 @@
+import xarray as xr
+import ee
+
+from .layer import Layer, get_utm_zone_epsg, get_image_collection
+
+
+class LandCoverGlad(Layer):
+    """
+    Attributes:
+        year: year used for data retrieval
+        spatial_resolution: raster resolution in meters (see https://github.com/stac-extensions/raster)
+    """
+
+    def __init__(self, year=2020, spatial_resolution=30, **kwargs):
+        super().__init__(**kwargs)
+        self.year = year
+        self.spatial_resolution = spatial_resolution
+
+    def get_data(self, bbox):
+        data = get_image_collection(
+            ee.ImageCollection(ee.Image(f'projects/glad/GLCLU2020/LCLUC_{self.year}')),
+            bbox,
+            self.spatial_resolution,
+            "GLAD Land Cover"
+        ).b1
+
+        return data
+
+
+class LandCoverSimplifiedGlad(Layer):
+    """
+    Attributes:
+        year: year used for data retrieval
+        spatial_resolution: raster resolution in meters (see https://github.com/stac-extensions/raster)
+    """
+
+    def __init__(self, year=2020, spatial_resolution=30, **kwargs):
+        super().__init__(**kwargs)
+        self.year = year
+        self.spatial_resolution = spatial_resolution
+
+    def get_data(self, bbox):
+        glad = LandCoverGlad(year=self.year, spatial_resolution=self.spatial_resolution).get_data(bbox)
+        # Copy the original data
+        data = glad.copy(deep=True)
+
+        # Assign values based on ranges and conditions
+        data = data.where(glad != 0, 0)  # If glad == 0, assign 0
+        # If glad between 1 and 24, assign 1
+        data = data.where((glad < 1) | (glad > 24), 1)
+        # If glad between 25 and 41, assign 2
+        data = data.where((glad < 25) | (glad > 41), 2)
+        # If glad between 42 and 48, assign 3
+        data = data.where((glad < 42) | (glad > 48), 3)
+        # If glad between 100 and 124, assign 4
+        data = data.where((glad < 100) | (glad > 124), 4)
+        # If glad between 125 and 148, assign 5
+        data = data.where((glad < 125) | (glad > 148), 5)
+        # If glad between 200 and 207, assign 6
+        data = data.where((glad < 200) | (glad > 207), 6)
+        # data = data.where(glad != 254, 6)  # Assign 6 if glad == 254
```

Comment:
@weiqi-tori There is a mistake in the original notebook. The category "254" corresponds to "ocean" and should be converted into "6" which is "water". The output data previously generated are following this rule. It seems that the line was commented by mistake. can you convert "254" into "6"?